.happy/terraform/modules/sfn_config/nextstrain.wdl

```diff
@@ -68,36 +68,37 @@ task nextstrain_workflow {
     )
 
     # set up ncov
-    mkdir -p /ncov/my_profiles/aspen /ncov/results
-    cd /ncov
-    git init
-    git fetch --depth 1 git://github.com/nextstrain/ncov.git
-    git checkout FETCH_HEAD
-    ncov_git_rev=$(git rev-parse HEAD)
+    mkdir -p ncov/my_profiles/aspen ncov/results
+    (cd ncov &&
+     git init &&
+     git fetch --depth 1 git://github.com/nextstrain/ncov.git &&
+     git checkout FETCH_HEAD
```

Comment:
what does this do?

Reply:
which line?

Reply:
`git checkout FETCH_HEAD`

Reply:
When one runs `git fetch`, it creates a temporary branch called "FETCH_HEAD" that represents the HEAD of what you just fetched.